src/main/java/net/sf/jabref/gui/keyboard/KeyBinding.java

```diff
@@ -13,6 +13,7 @@
     BACK("Back", Localization.lang("Back"), "alt LEFT"),
     CLEANUP(
             "Cleanup", Localization.lang("Cleanup entries"), "F8"),
+    CHECK_INTEGRITY("Check integrity", Localization.lang("Check integrity"), "ctrl F8"),
```

Comment:
In the `KeyBinding.java` only `Localization.lang` gets used. So I guess it must be right.